modules/client/examples/06-multisig-client/10-get-proposals.ts

```diff
@@ -0,0 +1,63 @@
+/* MARKDOWN
+### Loading the list of proposals (multisig plugin)
+*/
+import {
+  MultisigProposalListItem,
+  MultisigClient,
+  Context,
+  ContextPlugin,
+  IProposalQueryParams,
+  ProposalSortBy,
+  ProposalStatus,
+  SortDirection,
+} from "@aragon/sdk-client";
+import { contextParams } from "../00-client/00-context";
+
+// Create a simple context
+const context: Context = new Context(contextParams);
+// Create a plugin context from the simple context
+const contextPlugin: ContextPlugin = ContextPlugin.fromContext(context);
+// Create an multisig client
+const client = new MultisigClient(contextPlugin);
+
+const queryParams: IProposalQueryParams = {
+  skip: 0, // optional
+  limit: 10, // optional,
+  direction: SortDirection.ASC, // optional
+  sortBy: ProposalSortBy.POPULARITY, //optional
+  status: ProposalStatus.ACTIVE, // optional
+};
+
+const proposals: MultisigProposalListItem[] = await client.methods
```

Comment:
How do we specify what Plugin to list the proposals for? Same applies to everywhere

Reply:
There is a parameter that is `daoAddressOrEns` that we can use to filter by dao. I can add it to the examples or change it by the plugin address however in bot `AddresslistVoting` and `TokenVoting` plugins it uses the dao address to filter and if we decide to change it I would unify it

Reply:
Whatever the case, it has to be clear how to list the proposals of a certain DAO in the example :)